app/controllers/miq_policy_controller/alert_profiles.rb

```diff
@@ -353,10 +353,10 @@ def alert_profile_sync_provider(old_alerts = nil, new_alerts = nil)
       if old_alerts.nil? && new_alerts.nil?
         operation = :update_assignments
         old_alerts = new_alerts = @alert_profile.miq_alerts.collect(&:id)
-        assigned = @alert_profile.get_assigned_tos
       else
         operation = :update_alerts
       end
+      assigned = @alert_profile.get_assigned_tos
```

Comment:
why is this moved outside of the `if` part? is this fixing a bug or a logic change?